Tests/test_NCBIXML.py

```diff
@@ -1865,7 +1867,21 @@ def test_xml_2218L_rpsblast_001(self):
         # <Iteration_message>CONVERGED</Iteration_message>
         self.assertRaises(StopIteration, next, records)
         handle.close()
-
+    
+    def test_database_letters_deprecation(self):
+        with warnings.catch_warnings():
+            warnings.simplefilter("error", BiopythonDeprecationWarning)
+            filename = 'xml_2212L_blastp_001.xml'
+            with open(os.path.join("Blast", filename)) as handle:
+                try:
+                    records = NCBIXML.parse(handle)
+                except BiopythonDeprecationWarning as de:
+                    self.assertEqual(str(e),
+                                     "Accessing the .database_letters will" 
+                                     "be deprecated in the next" 
+                                     "version of Biopython.")
```

Comment:
Your string is missing spaces between "will" and "be", and "next" and "version". i.e. where you've split it up.